modular-docs-manual/files/TEMPLATE_ASSEMBLY_a-collection-of-modules.adoc

```diff
@@ -62,25 +58,26 @@ include::modules/TEMPLATE_CONCEPT_explaining_a_concept.adoc[leveloffset=+1]
 include::modules/TEMPLATE_PROCEDURE_doing_one_procedure.adoc[leveloffset=+2]
 // Add a blank line after each 'include::' statement.
 
-include::modules/TEMPLATE_PROCEDURE_reference-material.adoc[leveloffset=2]
+include::modules/TEMPLATE_PROCEDURE_reference-material.adoc[leveloffset=+2]
 
-[role="_additional-resources"]
 == Next steps
 
 * This section is optional.
-* Provide a bulleted list of links that contain instructions that might be useful to the user after they complete this procedure.
-* Use an unnumbered bullet (*) if the list includes only one step.
+* Provide information about the next steps the user might want to take.
+* This section can include text and links. 
 
 // If the last module included in your assembly contains an Additional resources section as well, check the appearance of the rendered assembly. If the two Additional resources sections might be confusing for a reader, consider consolidating their content and removing one of them.
+
 [role="_additional-resources"]
 == Additional resources
-
-* This section is optional.
-* Provide a bulleted list of links to other closely-related material. These links can include `link:` and `xref:` macros.
-* Use an unnumbered bullet (*) if the list includes only one step.
-
 ////
-Restore the context to what it was before this assembly.
```

Comment:
Similar to above, I would place it above the section title, not in between it and the bullet points.

Reply:
Really? The comment 'Restore the context to what it was before this assembly.'? Or did you mean something else?

Reply:
I either misclicked when adding this comment or it got moved with the latest changes. I was referring to the comment about deleting the Additional resources section if not needed.